config/MezaCoreExtensions.yml

```diff
@@ -329,12 +329,20 @@ list:
   - name: ContributionScores
     repo: https://github.com/wikimedia/mediawiki-extensions-ContributionScores.git
     version: "{{ mediawiki_default_branch }}"
-    legacy_load: true
```

Comment:
Contribution scores no longer needs to use the `legacy_load` attribute.

Plus, we added some more **Contribution Scores** configuration values - with comments.